e2etest/pkg/config/from_containers.go

```diff
@@ -0,0 +1,30 @@
+package config
+
+import (
+	frrk8sv1beta1 "github.com/metallb/frrk8s/api/v1beta1"
+	"go.universe.tf/e2etest/pkg/frr"
+	frrcontainer "go.universe.tf/e2etest/pkg/frr/container"
+	"go.universe.tf/e2etest/pkg/ipfamily"
+)
+
+func NeighborsForContainers(frrs []*frrcontainer.FRR, modify ...func(frr.Neighbor)) []frrk8sv1beta1.Neighbor {
+	res := make([]frrk8sv1beta1.Neighbor, 0)
+	for _, f := range frrs {
+		addresses := f.AddressesForFamily(ipfamily.IPv4)
+		ebgpMultihop := false
+		if f.NeighborConfig.MultiHop && f.NeighborConfig.ASN != f.RouterConfig.ASN {
+			ebgpMultihop = true
+		}
+
+		for _, address := range addresses {
+			neigh := frrk8sv1beta1.Neighbor{
+				ASN:          f.RouterConfig.ASN,
+				Address:      address,
+				Port:         f.RouterConfig.BGPPort,
+				EBGPMultiHop: ebgpMultihop,
```

Comment:
wdyt about adding the advertise prefixes and testing it at this point? I think this is "basic" enough to be in this pr but as you prefer